website/docs/developer/deployment-guide/deployment-with-cumulus.md

```diff
@@ -80,16 +80,20 @@ module "orca" {
   postgres_user_pw     = var.database_app_user_pw
 
   ## OPTIONAL
-  # database_port                        = 5432
-  # orca_ingest_lambda_memory_size       = 2240
-  # orca_ingest_lambda_timeout           = 600
-  # orca_recovery_buckets                = []
-  # orca_recovery_complete_filter_prefix = ""
-  # orca_recovery_expiration_days        = 5
-  # orca_recovery_lambda_memory_size     = 128
-  # orca_recovery_lambda_timeout         = 300
-  # orca_recovery_retry_limit            = 3
-  # orca_recovery_retry_interval         = 1
+  # database_port                                = 5432
+  # orca_ingest_lambda_memory_size               = 2240
+  # orca_ingest_lambda_timeout                   = 600
+  # orca_recovery_buckets                        = []
+  # orca_recovery_complete_filter_prefix         = ""
+  # orca_recovery_expiration_days                = 5
+  # orca_recovery_lambda_memory_size             = 128
+  # orca_recovery_lambda_timeout                 = 300
+  # orca_recovery_retry_limit                    = 3
+  # orca_recovery_retry_interval                 = 1
+  # sqs_delay_time                               = 0
+  # sqs_maximum_message_size                     = 262144
+  # staged_recovery_queue_message_retention_time = 432000
+  # status_update_queue_message_retention_time   = 777600
```

Comment:
I would personally vote we add units to the 3 names.

Reply:
yes I agree. It looks a bit confusing without knowing the variable description.

Reply:
@bhazuka should I make this change to the names? I know we merged the changes for https://bugs.earthdata.nasa.gov/browse/ORCA-157 last week and closed the card.

Reply:
Lets put a card in to update the names that way you can handle it across both. That way we can discuss prior to see if there are other variable names and or adjustments to make.

Reply:
created [ORCA-158](https://bugs.earthdata.nasa.gov/browse/ORCA-158) to address this in the future.